src/formats/cif.cpp

```diff
@@ -0,0 +1,205 @@
+//
+// Created by danny305 on May 3rd, 2021.
+//
+
+#include <iostream>
+#include <string>
+#include <vector>
+#include <stdexcept>
+#include <fstream>
+#include <filesystem>
+
+#include <gemmi/cif.hpp>
+#include <gemmi/mmcif.hpp>
+
+#define GEMMI_WRITE_IMPLEMENTATION
+#include <gemmi/to_cif.hpp>
+#undef GEMMI_WRITE_IMPLEMENTATION
+
+#include "chargefw2.h"
+#include "cif.h"
+#include "../structures/molecule_set.h"
+#include "../charges.h"
+#include "../config.h"
+
+
+namespace fs = std::filesystem;
+
+static const std::vector<std::string> atom_site_columns{
```

Comment:
Do we expect these columns are required/always present?

Reply:
if the cif file is from the PDB then these columns will always be present. These columns adhere from the mmCIF requirements. Maybe the solution would be to only write out the cif file if the input is a cif and it is a protein. This should be controlled in main.cpp.